modules/local/anndatar_convert.nf

```diff
@@ -0,0 +1,31 @@
+process ANNDATAR_CONVERT {
+
+    //
+    // This module uses the anndata R package to convert h5ad files in different formats
+    //
+
+    tag "${meta.id}"
+
+    label 'process_medium'
+
+    container "docker.io/fmalmeida/anndatar:dev" // TODO: Fix
```

Comment:
container is there now https://hub.docker.com/r/nfcore/anndatar/tags

Reply:
modified!